src/elements/ArtifactsTable/ArtifactsTable.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import './artifactstable.scss'
+import { formatDatetime } from '../../utils'
+const ArtifactsTable = ({ artifacts }) => {
+  const tableBody = artifacts.map((item, index) => {
+    const labels = item.labels.map((label, index) => {
+      var newLabel = label.replace(/=/, ': ')
```

Comment:
Instead of using `replace()` here, please change the `=` to `: `  in `parseKeyValues` method in src/utils/object.js

Reply:
Okay, I fixed it.